src/Tribe/Tickets_Handler.php

```diff
@@ -1385,7 +1385,7 @@ public function save_post( $post ) {
 	 * @return string
 	 */
 	public function save_form_settings( $post, $data = null ) {
-		// don't do anything on autosave, auto-draft, or massupdates
+		// don't do anything on autosave, auto-draft, or mass-updates
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: f22d49f7cd1dcf0948d656fbc2db8846</sub>